### PR TITLE
more doxygen

### PR DESCRIPTION
**Pull request by edwardhartnett**
@GeorgeGayno-NOAA this is the next file.

I'm sure to you this is obvious, but even looking at the GRIB template here: https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/grib2_temp3-30.shtml I can't be sure of what each parameter means.

There are "???" fields that need to be fixed in this PR.

Part of #97
Part of #111

### CODE REVIEW

File: src/ip_lambert_conf_grid_mod.F90 [hunk 1]
```diff
@@ -17,8 +22,8 @@ module ip_lambert_conf_grid_mod
   public :: ip_lambert_conf_grid
 
   type, extends(ip_grid) :: ip_lambert_conf_grid
-     real :: rlat1 !< ???
-     real :: rlon1 !< ???
+     real :: rlat1 !< La1― latitude of first grid point. GRIB2, Section 3.30, octet 39-42.
+     real :: rlon1 !< Lo1― longitude of first grid point. GRIB2, Section 3.30, octet 43-46.
      real :: rlati1 !< ???
```
Comment (GeorgeGayno-NOAA) [inline src/ip_lambert_conf_grid_mod.F90]: rlati1 - First latitude from the pole at which the secant cone cuts the sphere. GRIB2, Section 3, octets 66-69.
rlati2 - Second latitude from the pole at which the scant cone cuts the sphere. GRIB2, Section 3, octets 70-73.
orient - Longitude of meridian parallel to y-axis along which latitude increases at the latitude increases. GRIB2, Section 3, octets 52-55.

Comment (GeorgeGayno-NOAA) [reply src/ip_lambert_conf_grid_mod.F90]: dxs - x-direction grid length adjusted for scan mode. GRIB2, Section 3, octets 56-59.
dys - y-direction grid length adjusted for scan model. GRIB2, Section 3, octets 60-63.

Comment (GeorgeGayno-NOAA) [reply src/ip_lambert_conf_grid_mod.F90]: h - Hemisphere flag. 1-NH, minus 1-SH.

Comment (GeorgeGayno-NOAA) [reply src/ip_lambert_conf_grid_mod.F90]: irot - vector rotation flag. When "1", vectors are grid relative. When "0", vectors are earth relative. GRIB2, Section 3, octet 55.

Comment (edwardhartnett) [reply src/ip_lambert_conf_grid_mod.F90]: Thanks! I have filled those in.

Below them in the code there are these:

  INTEGER :: IROT !< ???
  REAL :: AN !< ???
  REAL :: DXS !< ???
  REAL :: DYS !< ???
  REAL :: H !< ???
  REAL :: RERTH !< ???

Comment (GeorgeGayno-NOAA) [reply src/ip_lambert_conf_grid_mod.F90]: AN - Cone factor
RERTH - Radius of the earth. GRIB2, Section 3, octets 15-30.

Note: AN is not an element in the GRIB2 template. So, there is no octet associated with it.

Also, a good reference for all the map projection equations used by IPLIB is this: https://doi.org/10.3133/pp1395
You might consider adding that to the wiki page or one of the readme files.

**Comment by edwardhartnett** [comment]
OK, lots of great detail going into the docs!

Let's get this batch of changes merged, and I'll continue in the next PR...

**Review by GeorgeGayno-NOAA — APPROVED** [review]

**Pull request merged**